### DISCUSSION TITLE
General question

### DISCUSSION THREAD

**Discussion by David-Mk** [bmoscon/cryptofeed#683] [General]
Hello! I have cloned the project, and followed instruction for set up. But is till don't get how shall I execute this one? I mean like how to run the whole thing? Thank you =)
In addition: perhaps adding some kind of a guide about the project (what is included inside, what folders are for etc.) would be useful, IMHO.

**Comment by bmoscon** [comment]
@David-Mk - have a look at the code under `examples/` - its a library, so you have to write some code to use it, this is not an application that you can just run out of the box.
